### ISSUE TITLE
Error creating obsgridDG in LIS_DAobservationsMod.F90

### ISSUE THREAD

**Issue by jvgeiger** [NASA-LIS/LISF#630]
LIS with ESMF 8 aborts at line 373 in subroutine create_obsdomain_objects in file LIS_DAobservationsMod.F90

```
[ERR] ESMF_DistGridCreate failed: obsgridDG Stopping.
```

The issue is deblklist is sized (2,2,LIS_npes).  However it is being assigned values like this:

```
367              deblklist(:,1,i+1) = (/stid/)
368              deblklist(:,2,i+1) = (/enid/)
```

This first dimension has a size of 2, but it is being assigned a single value.  Thus deblklist(2,:,:) contain uninitialized values.

Either deblklist should be sized (1,2,LIS_npes) or the assignment should be something like:`deblklist(:,1,i+1) = (/x,y/)`.

**Comment by bmcandr** [comment]
@jvgeiger, I believe Dan introduced a fix for this in the NUOPC Cap pull request that I'm currently reviewing.

It is in this commit in particular: https://github.com/NASA-LIS/LISF/pull/618/commits/58c6dddda54d6162ff749525b912de1721db05b3

**Comment by jvgeiger** [comment]
Thanks.  That does look like the correct fix.  The deblklist for obsgridDG looks like it should be (1,2,LIS_npes).  A few lines down, we have deblklist(2,2,LIS_npes) for gridEnsDG.  That block is consistent for a first dimension of size 2.

**Comment by dmocko** [comment]
I am debugging a separate issue, and added "-check all" to FFLAGS and FFLAGS77 in configure.lis

I see multiple messages also showing this error:

forrtl: severe (408): fort: (33): Shape mismatch: The extent of dimension 1 of array DEBLKLIST is 2 and the corresponding extent of array <RHS expression> is 1

Image              PC                Routine            Line        Source             
LIS                000000000841FDAF  Unknown               Unknown  Unknown
LIS                0000000000E1F326  lis_daobservation         368  LIS_DAobservationsMod.F90
LIS                0000000000E0C319  lis_daobservation         188  LIS_DAobservationsMod.F90
LIS                00000000074AEE82  retrospective_run          91  retrospective_runMod.F90
LIS                0000000001580712  lisinit_                  126  LIS_runmode_FTable.c
LIS                00000000045BB344  MAIN__                     65  lisdrv.F90
LIS                0000000000414352  Unknown               Unknown  Unknown
libc-2.22.so       00002AAAB45D7725  __libc_start_main     Unknown  Unknown
LIS                0000000000414269  Unknown               Unknown  Unknown

**Comment by bmcandr** [comment]
Fixed in #618.
